specutils/wcs/adapters/fitswcs_adapter.py

```diff
@@ -1,8 +1,10 @@
+import warnings
```

Comment:
I would suggest sticking with the regular `logging` setup that specutils already uses.